.evergreen/config.yml

```diff
@@ -716,7 +723,7 @@ buildvariants:
   - build-and-test-node
   - name: publish-packages
     distros:
-    - ubuntu1804-test
+    - ubuntu2004-large
```

Comment:
Was this needed for an upgrade of python versions?

I wonder if this is related to other failures I observed as part of the C++ driver on Ubuntu 18.04 giving an error about virtualenv not being defined: https://evergreen.mongodb.com/task/cxx_driver_mongohouse_ubuntu_test_mongohouse_bd929b61d61289383ef47ed3fd3493eabe9b6af9_20_12_01_16_18_35

Reply:
Actually, it is needed because for some reason the `ubuntu1804-*` images don't seem to have the virtualenv Python module installed on them.  It seemed simpler to just update to the more recent Ubuntu rather than write a BUILD ticket, especially since the underlying version of the OS is not important to the actions of the publish task.